src/include/homestore/index/index_table.hpp

```diff
@@ -247,7 +258,15 @@ class IndexTable : public IndexTableBase, public Btree< K, V > {
                           parent_node->node_id());
             return btree_status_t::not_found;
         }
-        BT_LOG(INFO, "Repairing node={} with last_parent_key={}", parent_node->to_string(),
+
+        // Get all current child ids
```

Comment:
please explain the algorithm here

Reply:
Added more comments.

The original child IDs serve as a supportive condition for deciding when to exit the loop. If a lower-level merge is committed while the parent-level merge is not, we may see children with keys beyond the original last key of the parent. In such case, we need to check if it's in the original children list to determine its parent.